client/src/api/metadata/__tests__/ApiClient.test.tsx

```diff
@@ -0,0 +1,252 @@
+import { test, describe } from 'vitest';
+import nock from 'nock';
+import { ApiClient } from '../ApiClient';
+import { SigMFMetadata, Annotation, CaptureSegment } from '@/Utils/sigmfMetadata';
+import { json } from 'react-router-dom';
+
+const account = 'gnuradio';
+const container = 'iqengine';
+const filePath = 'bluetooth';
+
```

Comment:
do we need to add a test to determine what the behaviour is when the metadata doesn't have annotations/captures?

Reply:
Added a new test